### PR TITLE
Ranking Criteria: Add repeat tags allowance

### PR DESCRIPTION
**Pull request by NoffyNoffeh**
Based off of the proposal found at https://osu.ppy.sh/community/forums/topics/1454937

### CODE REVIEW

**Review by cl8n — APPROVED** [review]
lgtm

File: wiki/Ranking_Criteria/en.md [hunk 1]
```diff
@@ -168,6 +168,7 @@ This category contains explicit allowance statements of concepts and rules that
 - **If a beatmap's track was contributed to by multiple artists, they may be listed with commas in between.** If there are 3 or more contributing artists and they are not part of one officially labelled group, `Various Artists` or other descriptive artist labels may be used instead.
 - **For remixes/covers, the original artist may be used in the artist field, as long as the title field is modified to clearly show that the song is remixed.** This marker should all be in parentheses and contain the remix/cover artist followed by descriptor.
 - **For live performances of a song, the title may include a `(Live Ver.)` marker.** Relevant details of the performance should be put in the beatmap description and tags, such as the date and location.
+- **Words may be repeated in tags, provided the repeated words are part of longer terms or phrases relevant to the beatmap.**
```
Comment (Walavouchey) [inline wiki/Ranking_Criteria/en.md:171]: from what i can tell from the thread, i think this should clarify what this allowance is about

```suggestion
- **Words already present in other metadata may be repeated in tags, provided the repeated words are part of longer terms or phrases relevant to the beatmap.**
```

Comment (cl8n) [reply wiki/Ranking_Criteria/en.md]: seems fine too, but remove "other"

Comment (NoffyNoffeh) [reply wiki/Ranking_Criteria/en.md]: In some cases the word may be repeated in tags themselves. Changed as suggested with the word other removed.

**Review by peppy — APPROVED** [review]

**Pull request merged**